app/pages/sections/catalogue-solutions/routes.test.js

```diff
@@ -160,13 +161,18 @@ describe('catalogue-solutions section routes', () => {
       })
     ));
 
-    it('should return the catalogue-solutions select recipient page if authorised', () => request(setUpFakeApp())
-      .get(path)
-      .set('Cookie', [mockAuthorisedCookie])
-      .expect(200)
-      .then((res) => {
-        expect(res.text.includes('Order item')).toBeTruthy();
-        expect(res.text.includes('data-test-id="error-title"')).toBeFalsy();
-      }));
+    it('should return the catalogue-solutions select recipient page if authorised', () => {
```

Comment:
change this test description 
`should return the catalogue-solutions order item page if authorised`